benchmark/src/benchmark/measure.clj

```diff
@@ -103,23 +128,93 @@
      :count n
      :observations (vec times)}))
 
-(defn get-measurements [{:keys [db-entity-counts config-name iterations] :as options}]
-  (->> (for [config (if (= config-name :all)
-                      c/db-configs
-                      (filter #(= (:config-name %) config-name) c/db-configs))
-             db-entities db-entity-counts
-             _ (range iterations)]
-         (measure-performance-full db-entities options config))
-       doall
-       (apply concat)
-       vec
-       (group-by :context)
-       (map (fn [[context group]]
-              (let [measurements (vec group)
-                    times (map :time measurements)]
-                (if (nil? context)
-                  nil
-                  {:context context
-                   :time (time-statistics times)}))))
-      
-       (remove nil?)))
+(defn get-measurements
+  ([options] (get-measurements options c/db-configs {}))
+  ([options configs] (get-measurements options configs {}))
+  ([{:keys [db-entity-counts] :as options} configs specified-fn]
+   (->> (for [cfg configs
+              entity-count db-entity-counts]
```

Comment:
Should be 
```
([{:keys [db-entity-counts db-samples] :as options} configs specified-fn]
   (->> (for [cfg configs
                  entity-count db-entity-counts
                  _ (range db-samples)]
```

And `db-samples` should be added as cli option. It will make sense to use this for query measurements if the focus is on rare queries that can't benefit from cached values.

Reply:
So `db-samples` should have the default value `1`.